src/RE.py

```diff
@@ -912,7 +912,7 @@ def pick_derivation(cognate_sets, statistics, only_with_mel):
     uniques = {}
     for cognate_set in cognate_sets:
         uniques[(correspondences_as_proto_form_string(cognate_set[0]), cognate_set[1])] = cognate_set
-    statistics.add_note(f'{len(uniques)} distinct reconstructions with distinct supporting forms')
+    statistics.add_note(f'{len(uniques)} cognate sets with distinct reconstructions and distinct supporting forms')
```

Comment:
I'm not sure if this is correct. Cognate sets appear with multiplicity and it's really the reconstructions that are being counted.

But I think we are generally inconsistent in our language.